.github/workflows/stale.yml

```diff
@@ -8,10 +8,10 @@ on:
 jobs:
   stale:
 
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/stale@v3
+    - uses: actions/stale@v6
```

Comment:
is there a 6?

Reply:
or did I write 6 before? I am on v5 anyway

Reply:
🤦🏻